### DISCUSSION TITLE
Minimum thickness in structure

### DISCUSSION THREAD

**Discussion by eflehder1** [leap71/PicoGK#51] [Coding for (Computational) Engineers]
Hi @leap71,
Is there a way to check for the minimum thickness within a PicoGK geometry?
Best,
Eric

**Comment by LinKayser** [comment]
Not really. It's not a simple problem. We usually know the minimum thickness from the way we design things. One visual way of checking it is to repeatedly look at the object after applying different offsets, possibly by outputting slices. Maybe there's an elegant geometric way that I am currently not thinking of.

Lin

**Reply by eflehder1** [reply]
Thank you @LinKayser, I tried an approach of iterating through the surface voxels then casting a ray up to the other side in all translational degrees of freedom and get the thickness distribution. It didnt work too well so not posting it, for now Im just checking this manually and also being more careful to keep track of this in the design process as you suggest.
